client/src/pages/login/login/login.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './login.css';
+import { Link } from 'react-router-dom' //Links Library from React Router
+
+function Login() {
+  return (
+  	<div className="App-login" id="Login">   {/* This code basically puts "This is the login screen" in big words*/}
+	  	<div className="container">			 {/* Along with tester links at the bottom to each respective dash */}
+		  	<div className="login">
+		  	<h1>This is the login screen</h1>   {/* Big words of "this is the login screen" in middle of page */}
+		  	</div>
+		  	<div className = "links">
+		  	<span>
+		  		<Link to = '/login/app'>Applicant</Link>   {/* React Router link to Applicants Dash */}
+		  		<a>&nbsp;&nbsp;</a>
```

Comment:
Should use CSS styling here, there's a reason HTML tries to restrict padding with spaces.

Reply:
got it! I implemented the same result using the `width` selector and using percentages to implement the 4 elements. Therefore 25%.